src/librustdoc/display/maybe.rs

```diff
@@ -0,0 +1,17 @@
+use std::fmt::{self, Display};
```

Comment:
Please make one `display.rs` file instead of having a lot of small files around. Makes the reading more tricky than necessary in this case.